src/components/Table/HeaderCell.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react'
+
+import Icon from '../Icon'
+import { SortableCellUI, HeaderCellUI } from './styles/Table.css'
+import { TABLE_CLASSNAME } from './Table'
+
+import { HeaderCellProps } from './types'
+
+export default class HeaderCell extends React.PureComponent<HeaderCellProps> {
+  render() {
+    const { column, isLoading, sortedInfo } = this.props
+
+    return (
+      <HeaderCellUI
+        className={`${TABLE_CLASSNAME}__HeaderCell`}
+        align={column.align}
+        cellWidth={column.width}
+        aria-sort={
+          sortedInfo &&
+          sortedInfo.columnKey === column.columnKey &&
+          sortedInfo.order
+            ? sortedInfo.order
+            : 'none'
+        }
+        onClick={() => {
```

Comment:
@tinkertrain Can this be refactored to be a class method instead of an inline function? 😊

Reply:
It certainly can!

Reply:
Changed!